packages/stitch/src/stitchSchemas.ts

```diff
@@ -202,13 +207,25 @@ function applySubschemaConfigTransforms(
     ? subschemaOrSubschemaConfig
     : { schema: subschemaOrSubschemaConfig };
 
-  const newSubschemaConfig = subschemaConfigTransforms.reduce((acc, subschemaConfigTransform) => {
-    return subschemaConfigTransform(acc);
-  }, subschemaConfig);
+  let transformedSubschemaConfigs: Array<SubschemaConfig> = [subschemaConfig];
+  subschemaConfigTransforms.concat(subschemaConfigTransformerPresets).forEach(subschemaConfigTransform => {
+    const mapped: Array<SubschemaConfig | Array<SubschemaConfig>> = transformedSubschemaConfigs.map(ssConfig =>
+      subschemaConfigTransform(ssConfig)
+    );
+
+    transformedSubschemaConfigs = mapped.reduce(
+      (acc: Array<SubschemaConfig>, configOrList: SubschemaConfig | Array<SubschemaConfig>) => {
+        if (Array.isArray(configOrList)) {
+          return acc.concat(configOrList as Array<SubschemaConfig>);
+        }
+        acc.push(configOrList as SubschemaConfig);
+        return acc;
+      },
+      []
+    ) as Array<SubschemaConfig>;
```

Comment:
This is an incredibly verbose implementation of `Array.prototype.flat()`. Unfortunately, the repo's Node targets don't allow for the built-in method it appears.